pkg/controller/controller_instance.go

```diff
@@ -605,6 +654,11 @@ func (c *controller) pollInstance(serviceClass *v1alpha1.ServiceClass, servicePl
 				message,
 			)
 		}
+
+		err = c.continuePollingInstance(instance)
+		if err != nil {
+			return err
+		}
 		return fmt.Errorf("last operation not completed (still in progress) for %v/%v", instance.Namespace, instance.Name)
```

Comment:
We don't need this error anymore, do we? Since we're calling `continuePollingInstance` to re-add the key to the polling queue, we could return `nil` here and the instance will still be reprocessed.